src/main/java/com/mathworks/ci/RunMatlabTestsStep.java

```diff
@@ -0,0 +1,161 @@
+package com.mathworks.ci;
+
+/**
+ * Copyright 2019-2020 The MathWorks, Inc.
+ *  
+ */
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.commons.io.FilenameUtils;
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor;
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+import com.google.common.collect.ImmutableSet;
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+
+public class RunMatlabTestsStep extends Step {
+    
+    private String testResultsPdf;
+    private String testResultsTAP;
+    private String testResultsJUnit;
```

Comment:
What is the capitalization style you should be using? It seems to switch between only capitalizing the first letter in names like "Matlab" and "Pdf", to using all caps in "TAP", to using caps that match the brand in "JUnit". I don't think any is necessarily wrong but you should be consistent.

Reply:
I had picked them as per my last naming pass. I TAP & JUnit are consistent with azure pipeline i shall change the Pdf to PDF